database/migrations/2020_12_14_091314_create_port_groups_table.php

```diff
@@ -21,8 +21,6 @@ public function up()
             $table->unsignedInteger('port_group_id')->unsigned()->index();
             $table->unsignedInteger('port_id')->unsigned()->index();
             $table->primary(['port_group_id', 'port_id']);
-            $table->foreign('port_group_id')->references('id')->on('port_groups')->onDelete('CASCADE');
-            $table->foreign('port_id')->references('port_id')->on('ports')->onDelete('CASCADE');
         });
```

Comment:
Best practice is to have one table migration per migration file, I read somewhere Taylor said that, and it makes sense either way

Reply:
true, I can split it out into 4 migrations.